packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -3755,2133 +3905,2220 @@ describe('ReactDOMFloat', () => {
+          } else {
+            expect(mockError).not.toHaveBeenCalled();
+          }
+        } finally {
+          console.error = originalConsoleError;
+        }
       });
 
-      expect(getMeaningfulChildren(document)).toEqual(
-        <html>
-          <head>
-            <link
-              rel="stylesheet"
-              href="foo"
-              data-precedence="default"
-              crossorigin="anonymous"
-              media="all"
-              integrity="somehash"
-              referrerpolicy="origin"
-              data-foo={'"quoted"'}
-              nonstandardattr="attr"
-              properlyformattednonstandardattr="attr"
-            />
-            <link
-              rel="preload"
-              as="style"
-              href="foo"
-              precedence="default"
-              crossorigin="anonymous"
-              media="all"
-              integrity="somehash"
-              referrerpolicy="origin"
-              data-foo={'"quoted"'}
-              nonstandardattr="attr"
-              properlyformattednonstandardattr="attr"
-            />
-          </head>
-          <body>
-            <div>loading...</div>
-          </body>
-        </html>,
-      );
-      if (__DEV__) {
-        // The way the test is currently set up the props that would warn have already warned
-        // so no new warnings appear. This is really testing the same code pathway so
-        // exercising that more here isn't all that useful
-        expect(mockError).toHaveBeenCalledTimes(0);
-      } else {
-        expect(mockError).not.toHaveBeenCalled();
-      }
-    } finally {
-      console.error = originalConsoleError;
-    }
-  });
-
-  // @gate enableFloat
-  it('boundary style resource dependencies hoist to a parent boundary when flushed inline', async () => {
-    function BlockedOn({text, children}) {
-      readText(text);
-      return children;
-    }
-    await actIntoEmptyDocument(() => {
-      const {pipe} = renderToPipeableStream(
-        <html>
-          <head />
-          <body>
-            <div>
-              <Suspense fallback="loading A...">
-                <BlockedOn text="unblock">
-                  <AsyncText text="A" />
-                  <link rel="stylesheet" href="A" precedence="A" />
-                  <Suspense fallback="loading AA...">
-                    <AsyncText text="AA" />
-                    <link rel="stylesheet" href="AA" precedence="AA" />
-                    <Suspense fallback="loading AAA...">
-                      <AsyncText text="AAA" />
-                      <link rel="stylesheet" href="AAA" precedence="AAA" />
-                      <Suspense fallback="loading AAAA...">
-                        <AsyncText text="AAAA" />
-                        <link rel="stylesheet" href="AAAA" precedence="AAAA" />
+      // @gate enableFloat
+      it('boundary style resource dependencies hoist to a parent boundary when flushed inline', async () => {
```

Comment:
this test fails on the browser version @gnoff, I'm not sure if it's a bug on Float or because of the way I wrapped the streams